src/main/java/com/manardenza/servise/UserService.java

```diff
@@ -0,0 +1,24 @@
+package com.manardenza.servise;
+
+import com.manardenza.dao.UserDaoImpl;
+import com.manardenza.entity.User;
+
+public class UserService {
+
+    private static UserService instance;
+    private UserDaoImpl userDao;
+
+    private UserService() {
+        this.userDao = UserDaoImpl.getInstance();
+    }
+
+    public static UserService getInstance() {
+        if (instance == null) {
+            instance = new UserService();
+        }
+        return instance;
+    }
+
+    public void registerUser(User userToRegistration) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)